src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -324,7 +324,7 @@
 				.description("Called when a player starts or stops sneaking. Use <a href='conditions.html#CondIsSneaking'>is sneaking</a> to get whether the player was sneaking before the event was called.")
 				.examples("# make players that stop sneaking jump",
 						"on sneak toggle:",
-						"	player was sneaking",
+						"	player is sneaking",
```

Comment:
player was sneaking is valid syntax. Why remove it?

Reply:
It is related to this [comment](https://github.com/SkriptLang/Skript/issues/2705#issuecomment-659087051) specifically

Reply:
I stand corrected then! All looks good to me.